pkg/knuu/knuu.go

```diff
@@ -82,6 +81,11 @@ func New(ctx context.Context, opts Options) (*Knuu, error) {
 	return k, nil
 }
 
+func DefaultTestScope() string {
+	t := time.Now()
+	return fmt.Sprintf("%s-%03d", t.Format("20060102-150405"), t.Nanosecond()/1e6)
+}
+
```

Comment:
We shouldn't put test helpers in production code. 
Test helpers can be put in the `e2e` package.

Reply:
agreed